mobile/examples/Xamarin/VisionSample/VisionSample/SkiaSharpImageProcessor.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using Microsoft.ML.OnnxRuntime.Tensors;
+using SkiaSharp;
+
+namespace VisionSample
+{
+    public class SkiaSharpImageProcessor<TPrediction, TTensor> : IImageProcessor<SKBitmap, TPrediction, TTensor>
+    {
+        protected virtual SKBitmap OnPreprocessSourceImage(SKBitmap sourceImage) => sourceImage;
+        protected virtual Tensor<TTensor> OnGetTensorForImage(SKBitmap image) => throw new NotImplementedException();
+        protected virtual void OnPrepareToApplyPredications(SKBitmap image, SKCanvas canvas) { }
```

Comment:
Assume this should be `OnPrepareToApplyPredictions` ?

predications -> predictions?